src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -246,40 +247,41 @@ class Filter extends Component<Props> {
     const { filterState } = this.props
     return (
       <Flex justifyContent={["space-between", "flex-end"]} alignItems="center">
-        <SelectSmall
-          options={[
-            {
-              value: "-decayed_merch",
-              text: "Default",
-            },
-            {
-              value: "-partner_updated_at",
-              text: "Recently updated",
-            },
-            {
-              value: "-published_at",
-              text: "Recently added",
-            },
-            {
-              value: "-year",
-              text: "Artwork year (desc.)",
-            },
-            {
-              value: "year",
-              text: "Artwork year (asc.)",
-            },
-          ]}
-          selected={filterState.state.sort}
-          title="Sort"
-          onSelect={sort => {
-            return filterState.setSort(sort)
-          }}
-        />
+        <Box mt={-0.5}>
```

Comment:
Fixes alignment issue with new `SmallSelect` component, recently dropped into layout. 

<img width="405" alt="Screen Shot 2019-06-04 at 5 45 37 PM" src="https://user-images.githubusercontent.com/236943/58924131-70431780-86f7-11e9-9b2d-e65db9d2f5e7.png">

Reply:
Is this an indication that `SmallSelect` has a positive margin or padding that it shouldn't?

Reply:
I don't _think_ it does (cc @oxaudo) but rather points to some in-general layout issues in the filter itself that need correcting